.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Doc issue
+about: Create an issue to help us improve
+
+---
+
+**Before you enter an issue**
+
+- If the issue is a simple typo or similar correction, you can just submit a PR. See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions, if needed.
+- If you are are struggling with a tutorial, check your code against the completed sample for differences.
+- If the issue is a general support question rather than a docs issue, consider a forum site instead of a docs issue.
```

Comment:
It might be good to mention that if the issue concerns site design, it should be done at MicrosoftDocs/feedback.